proof/refine/ARM/Invariants_H.thy

```diff
@@ -3868,12 +3890,16 @@ private lemma ko_at_defn_ko_wp_at':
   by (clarsimp simp: ko_at'_defn_def obj_at'_real_def
                      ko_wp_at'_def project_inject)
 
-method normalise_obj_at' =
+(* VER-1364: normalise_obj_at' sometimes doesn't normalise ob_at' unless you use it twice. *)
```

Comment:
I probably don’t see the point of this comment. Can you be a bit more clear? How does the JIRA issue relate to what is written here?

Reply:
The point is so when someone comes along later and sees this weird repeated method, they know _why_ it's a weird repeated method: for some reason it doesn't always do what we want it to do, and it's a known issue, and if they want more details they can look up the issue.

Reply:
If there is something that still needs to be fixed, could you please label it with “FIXME RT”? If the JIRA issue has more details of the issue, you can also say “see VER-XXX for details” or something like that?